src/utils/map-controls.js

```diff
@@ -86,6 +86,11 @@ export default class MapControls {
       srcEvent.ctrlKey || srcEvent.shiftKey);
   }
 
+  isRightButtonPressed(event) {
+    const {srcEvent: {which, buttons}} = event;
+    return buttons === undefined ? (which === 3) : Boolean(buttons & 2);
```

Comment:
Can we add a comment to indicate what the logic here is? i.e. what is `3` (assuming it's the right mouse code), and what `buttons & 2` does.

Reply:
Actually, do we need to use `&` here?

Reply:
[buttons](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons) is a sum of the values of all active buttons:

Reply:
I see, thanks! It's definitely worth leaving a comment then, especially for the `&` and how it's an accurate way of detecting the right mouse press.